gulpfile.js

```diff
@@ -92,6 +92,17 @@ gulp.task('svg-individual', function () {
 gulp.task('svg', ['svg-sprite', 'svg-individual']);
 
 
+// BUILD DESIGN SYSTEM STYLES
+gulp.task('build-styles', function () {
+	return gulp.src('dist/styles.scss')
+		.pipe(scss())
+		.pipe(autoprefixer({
+			browsers: ['last 3 versions'],
```

Comment:
I'm not sure. This just matches what we're doing for reference page. Someone with better knowledge on why the config options were chosen for both RP and Mercury would be able to make a call.